scripts/mbedtls_dev/asymmetric_key_data.py

```diff
@@ -41,7 +41,7 @@ def construct_asymmetric_key_data(src) -> Dict[str, Dict[int, bytes]]:
     'ECC(PSA_ECC_FAMILY_SECP_K1)': {
         192: ("297ac1722ccac7589ecb240dc719842538ca974beb79f228",
               "0426b7bb38da649ac2138fc050c6548b32553dab68afebc36105d325b75538c12323cb0764789ecb992671beb2b6bef2f5"),
-        224: ("0024122bf020fa113f6c0ac978dfbd41f749257a9468febdbe0dc9f7e8",
+        224: ("24122bf020fa113f6c0ac978dfbd41f749257a9468febdbe0dc9f7e8",
```

Comment:
Note: this might not be the correct fix after all, see https://github.com/Mbed-TLS/mbedtls/pull/4076#discussion_r1440275352